src/tokenizer.ts

```
@@ -0,0 +1,585 @@
+  state: EscapeMode,
+  character: string,
+  tokens: Token[],
+): void {
+  switch (character) {
+    case "{":
+    case "\\":
+      tokens.push(new Token(TokenType.Literal, character, flushRange(state)));
+      break;
+    default:
+      tokens.push(
+        new Token(TokenType.Literal, "\\" + character, flushRange(state)),
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)